src/app/services/mail/mail.types.ts

```diff
@@ -71,9 +71,14 @@ export type SubmissionToAdminHtmlData = {
   appName: string
 }
 
+export type AutoreplyHtmlDefaultBody = {
```

Comment:
this type is not meant to be used outside right? so you don't need to export it. principle of encapsulation - hide the internal representation of data in this module; only expose what is necessary for other modules to work with it